cuML/src/glm/ridge.h

```diff
@@ -162,23 +163,14 @@ void ridgeFit(math_t *input, int n_rows, int n_cols, math_t *labels,
 	}
 
 	if (fit_intercept) {
-		postProcessData(input, n_rows, n_cols, labels, coef, intercept,
-				mu_input, mu_labels, norm2_input, fit_intercept, normalize,
-				cublas_handle, cusolver_handle);
-
-		if (normalize) {
-			if (norm2_input != NULL)
-				cudaFree(norm2_input);
-		}
-
-		if (mu_input != NULL)
-			cudaFree(mu_input);
-		if (mu_labels != NULL)
-			cudaFree(mu_labels);
+            postProcessData(input, n_rows, n_cols, labels, coef, intercept, mu_input.data(),
+                            mu_labels.data(), norm2_input.data(), fit_intercept, normalize,
+                            cublas_handle, cusolver_handle);
 	} else {
 		*intercept = math_t(0);
 	}
-
+    CUDA_CHECK(cudaStreamSynchronize(s_));
+    CUDA_CHECK(cudaStreamDestroy(s_));
```

Comment:
`cudaStreamDestroy` is not needed if the handle is passed in.

Reply:
Done.